porting/npl/freertos/include/nimble/nimble_npl_os.h

```diff
@@ -128,7 +122,7 @@ ble_npl_eventq_run(struct ble_npl_eventq *evq)
 {
     struct ble_npl_event *ev;
 
-    ev = ble_npl_eventq_get(evq);
+    ev = ble_npl_eventq_get_tmo(evq, BLE_NPL_TIME_FOREVER);
```

Comment:
it probably does not build as ble_npl_eventq_get_tmo is removed

Reply:
it builds because I removed `ble_npl_eventq_get` instead from this NPL :-)
but mesh probably does not build on FreeRTOS since it uses `ble_npl_eventq_get`.
will fix.